README.md

```diff
@@ -1 +1,73 @@
-# Bayan_Ali--project
\ No newline at end of file
+## Movie Night website
+
+## Description
+
+This is a web app application for people whose interested in movies and pssionate so much about their movies characters and stuff.
+Make your self home, and pick what ever movie that comes to your mind, and get the full access to its details consisting the crew, staff , rate and some related links, all in one place.
+
+## Live Link 
+You can check the website [Here](https://bayan-ali-movies-expressjs.herokuapp.com/).
+
+
+## API we used
+ - [Movies API](https://www.themoviedb.org/documentation/api)
+
+
+## User journey
+
+- The user's journey starts with opening our website and seeing our landing page. The user will see a small description and a button to move. 
+
+
+- When the user clicks on the let's start button, he'll move to the movies page where he can see a bunch of popular movies that are highly recommended to wath, besides it's containing a search bar, where he can write his favorite movie's name. 
+
+- when the user starts writing a movie name, will display a list of movies depending on the search value.
+
+
+
+
+## Technologies we used
+
+
+- HTML
+
+- CSS
+
+- JS | DOM Manipulations: used to connect the pages with each others and make the website interactive.
+
+- Fetch: used to establish connection with the APIs.
+
+- Express.js: to run the server, and serving the user with the desired data fetched from API.
+
+##  How to install the project locally:
```

Comment:
nice readme, but the installation guide needs more details like `install dependencies using npm i` `create .env file in the root ...` and `run the dev server using npm run dev` ...etc